### PR TITLE
build: added image-picker feature

### PR DESCRIPTION
**Pull request by AudreyKj**
added UI for image picker (in NewMessageForm.js) and util function (imagePicker.js) returning base64 url of image 

89718658-17d96b00-d9c9-11ea-93d6-139ecf01fca2

### CODE REVIEW

**Comment by DarkSouL11** [comment]
@AudreyKj Can you please add screenshots for the UI?

File: client/src/components/Messaging/NewMessageForm.js [hunk 1]
```diff
@@ -22,6 +24,12 @@ export const NewMessageForm = ({ handleSubmit, text, setText }) => {
         onChange={(e) => setText(e.target.value)}
         value={text}
       />
+      <div className={styles.imagePickerContainer}>
+        <label className={styles.imagePickerLabel} forhtml="file-upload">
```
Comment (DarkSouL11) [inline client/src/components/Messaging/NewMessageForm.js:28]: I think you wanted to use `htmlFor` here. I this case that also should not be needed as input is a child of label

File: client/src/utils/imagePicker.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+    let reader = new FileReader();
+    let file = e.target.files[0];
+
+    reader.readAsDataURL(file);
+
+    reader.onload = (e) => {
+      resolve(e.target.result);
+    };
+
+    reader.onload = resolve;
```
Comment (muke1908) [inline client/src/utils/imagePicker.js:13]:
```suggestion

```

**Comment by AudreyKj** [comment]
@muke1908 @DarkSouL11 thanks for reviewing it! I have made the necessary edits!

**Pull request merged**